common/app/common/commercial/hosted/HostedGalleryPage.scala

```diff
@@ -22,6 +22,7 @@ case class HostedGalleryPage(
 
   val pageTitle: String = s"Advertiser content hosted by the Guardian: $title - gallery"
   val twitterText = shareText.getOrElse(if(standfirst.length < 136) standfirst else title) + " #ad"
+  val imageUrl = images.head.url
```

Comment:
Hmm I hadn't thought about that. Though a gallery without any images isn't really a gallery is it?! 😸